### ISSUE TITLE
Support Android SDK version 26

### ISSUE THREAD

**Issue by mayyuen318** [SkygearIO/skygear-SDK-Android#158]
When I use compiled SDK version 26, an error is shown:

```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute meta-data#android.support.VERSION@value value=(26.0.0-alpha1) from [com.android.support:design:26.0.0-alpha1] AndroidManifest.xml:27:9-38
  	is also present at [com.android.support:support-v13:25.3.1] AndroidManifest.xml:27:9-31 value=(25.3.1).
  	Suggestion: add 'tools:replace="android:value"' to <meta-data> element at AndroidManifest.xml:25:5-27:41 to override.
```

**Comment by AK-007** [comment]
Maybe build.gradle should have two changes:
 compileSdkVersion 26
 buildToolsVersion "26.0.1"

**Comment by mayyuen318** [comment]
My original gradle looks like this
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 26
    buildToolsVersion "26.0.1"
    defaultConfig {
        applicationId "test.testwithskygear2"
        minSdkVersion 23
        targetSdkVersion 26
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile 'com.android.support:appcompat-v7:26.+'
    compile 'com.android.support:design:26.+'
    testCompile 'junit:junit:4.12'

    // other dependencies
    compile 'io.skygear:skygear:+'
}

```
And the above error still ><. 


Using version 25, it works. Here is the code
```
apply plugin: 'com.android.application'

android {
    compileSdkVersion 25
    buildToolsVersion '25.0.0'
    defaultConfig {
        applicationId "test.testwithskygear"
        minSdkVersion 23
        targetSdkVersion 25
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    compile fileTree(dir: 'libs', include: ['*.jar'])
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.0', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile 'com.android.support:appcompat-v7:25.+'
    compile 'com.android.support:design:25.+'
    testCompile 'junit:junit:4.12'

    // other dependencies
    compile 'io.skygear:skygear:+'


}

```

**Comment by aldocano** [comment]
Please show Project gradle file.

**Comment by aldocano** [comment]
make sure it is it contains 

allprojects {
    repositories {
        jcenter()
        maven { url 'https://jitpack.io' }
        maven { url "https://maven.google.com" }
    }
}

**Comment by mhdsmr2** [comment]
@aldocano I have updated the Android SDK version, please merge the pull request.

**Comment by mhdsmr2** [comment]
Pull request https://github.com/SkygearIO/skygear-SDK-Android/pull/159 is mine. Please merge it.

**Comment by aldocano** [comment]
@mhdsmr2  I just guided him to fix it, im not the one who does the merge. By the way nice pull request.

**Comment by mhdsmr2** [comment]
So, @mayyuen318 will you do the merge, please?

**Comment by chpapa** [comment]
thanks @mhdsmr2 and @aldocano. I have assigned one of the maintainer @rickmak to merge the PR

**Comment by mhdsmr2** [comment]
@chpapa. Thanks a lot friend :)

**Comment by vankasaisumanth** [comment]
I am using sdk version 26. when I run the gradle.app to sync, it says "failed to resolve the recyclerview-v7:26.+" then I have used version 25 still the only recyclerview error persists saying "failed to resolve the recyclerview-v7:27.0.3"........(27.0.3 because my android studio say recommended version/ below that doesnot support). Even I have tried latest 27.1.1 but no use.


Need some help.
